### PR TITLE
Rename dropdown to more closely match existing and base elements

### PR DESCRIPTION
**Pull request by wishdasher**
I originally named the components closely to what they were called in the design. I realized that those names don't match the existing components or the basic HTML elements that the components correspond to, so I'm trying to improve that.

Test plan:
The snapshot of this package doesn't change with this diff. Searched through the codebase for all instances of the names I changed.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit d312bf8e4d3c7b838e430d1c34d425efc2757221

https://deploy-preview-238--wonder-blocks.netlify.com

File: packages/wonder-blocks-dropdown/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-// import SingleSelectMenu from "./components/single-select-menu.js";
-// import MultiSelectMenu from "./components/multi-select-menu.js";
-//
-// export {ActionMenu, SingleSelectMenu, MultiSelectMenu};
+// @flow
+import ActionMenu from "./components/action-menu.js";
+import SingleSelect from "./components/single-select.js";
+import MultiSelect from "./components/multi-select.js";
+
+export {ActionMenu, SingleSelect, MultiSelect};
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/index.js]: Yay!  Can you add a test for the exports of this file similar to what we do in https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-typography/index.test.js?

File: packages/wonder-blocks-dropdown/components/action-menu.js [hunk 2]
```diff
@@ -180,22 +177,27 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
         const opener = (
             <ActionMenuOpener
                 disabled={disabled}
-                onClick={() => this.toggleMenu()}
+                onClick={() => this.handleOpenChanged(!open)}
+                ref={(node) =>
+                    (this.openerElement = ((ReactDOM.findDOMNode(
+                        node,
+                    ): any): Element))
```
Comment (jeresig) [inline packages/wonder-blocks-dropdown/components/action-menu.js]: Woah! Oh Flow... 😬

File: packages/wonder-blocks-dropdown/components/multi-select.js [hunk 3]
```diff
@@ -93,15 +96,9 @@ export default class MultiSelectMenu extends React.Component<Props, State> {
...
-        this.setState((prevState) => ({
-            open: !prevState.open,
-        }));
-    }
-
-    handleClose() {
+    handleOpenChanged(open: boolean) {
         this.setState({
-            open: false,
+            open: open,
```
Comment (jeresig) [inline packages/wonder-blocks-dropdown/components/multi-select.js]: Can also do: `this.setState({open});`

**Pull request merged**